vm-deployment/vm.bicep

```
@@ -0,0 +1,282 @@
+/*
+Virtual machine deployment.
+*/
+
+// Parameters for resource names
+param virtualMachines string = 'iagora'
+param publicIPAddresses string = 'iagora-ip'
+param virtualNetworks string = 'iagora-vnet'
+param networkInterfaces string = 'iagora731_z1'
+param networkSecurityGroups string = 'iagora-nsg'
+
+// SSH public key parameter
+@description('The SSH public key to use for authentication.')
+param sshPublicKey string = 'ssh-rsa AAAAB3...your-public-key... user@domain'
```

```suggestion
param sshPublicKey string
```
This default value is not a valid public SSH key, right?

Could we move this example into the description and make this parameter required?